<launch>
  <arg name="rob_id" default="thorvald_001"/>

    <!-- AMCL -->
    <include file="$(find uol_cmp9767m_tutorial)/launch/amcl.launch">
      <arg name="robot_name" value="$(arg rob_id)"/>
    </include>
    <!-- Navigation Stack (move_base)-->
    <include file="$(find uol_cmp9767m_tutorial)/launch/move_base.launch">
      <arg name="robot_name" value="$(arg rob_id)"/>
    </include>

    <!-- Weeding nodes -->
    <group ns="$(arg rob_id)">
      <node pkg="rp_pack" type="plant_discrimination.py" name="plant_discriminator" args="$(arg rob_id)"/>
      <node pkg="rp_pack" type="camera_geometry.py" name="camera_converter" args="$(arg rob_id)"/>
    </group>
</launch>
